torch/utils/data/dataloader.py

```diff
@@ -45,7 +54,7 @@ def _worker_loop(dataset, index_queue, data_queue, collate_fn):
             data_queue.put((idx, samples))
 
 
-def _pin_memory_loop(in_queue, out_queue, done_event):
+def _process_loop(in_queue, out_queue, done_event, pin_memory):
```

Comment:
`_process_loop` doesn't really tell you what it does. Maybe `_worker_manager_loop`?